ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -605,3 +605,68 @@ periodics:
     - configMap:
         name: prow-job-cluster-launch-e2e-azure
       name: job-definition
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: periodic-ci-openshift-openshift-master-azure-e2e-bushslicer
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --git-ref=openshift/openshift-azure@master
+      - --secret-dir=/usr/local/e2e-secrets/azure
+      - --target=e2e-azure-bushslicer
```

Comment:
I think you are missing a couple of other targets here for building the rest of the osa images (sync, azure-controllers, etcdbackup). See `periodic-ci-openshift-openshift-azure-conformance` as an example.

Reply:
What happens if these aren't created?  Can it pull latest?  Are these required to be built each time?  I will add them if required.

Reply:
Good questions, they may not be required for periodics, let's merge and see it if breaks.

Reply:
I have updated the PR to reflect that we want sync and controllers.  I pass those locally so I've updated the job here with those parameters.